packages/plugins/Scale/src/store/index.ts

```diff
@@ -76,6 +72,23 @@ export const makeStoreModule = () => {
     },
     getters: {
       ...generateSimpleGetters(getInitialState()),
+      zoomOptions: (_, __, ___, rootGetters) => {
+        const options =
+          rootGetters.configuration?.scale?.options ||
```

Comment:
This is probably related to the question in the PR description: 

Why is there a separate option if all the scales are already coming from one place (the fallback `rootGetters.configuration.options`)? I don't really see the necessity